packages/whale-api-client/__tests__/api/consortium.test.ts

```diff
@@ -188,3 +190,340 @@ describe('getAssetBreakdown', () => {
+  const startFlags: StartFlags[] = [{ name: 'regtest-minttoken-simulate-mainnet', value: 1 }]
+  const txIds: string[] = []
+
+  beforeAll(async () => {
+    await tGroup.start({ startFlags })
+    await service.start()
+    await alice.container.waitForWalletCoinbaseMaturity()
+
+    app = await createTestingApp(alice.container)
+
+    await setup()
+  })
+
+  afterAll(async () => {
+    try {
+      await service.stop()
+    } finally {
+      await stopTestingApp(tGroup, app)
+    }
+  })
+
+  async function setGovAttr (ATTRIBUTES: object): Promise<void> {
+    const hash = await alice.rpc.masternode.setGov({ ATTRIBUTES })
+    expect(hash).toBeTruthy()
+    await alice.generate(1)
+    await tGroup.waitForSync()
+  }
+
+  async function setMemberInfo (tokenId: string, memberInfo: Array<{ id: string, name: string, backingId: string, ownerAddress: string, mintLimit: string, mintLimitDaily: string }>): Promise<void> {
+    const infoObjs: { [key: string]: object } = {}
+
+    memberInfo.forEach(mi => {
+      infoObjs[mi.id] = {
+        name: mi.name,
+        ownerAddress: mi.ownerAddress,
+        backingId: mi.backingId,
+        mintLimitDaily: mi.mintLimitDaily,
+        mintLimit: mi.mintLimit
+      }
+    })
+
+    return await setGovAttr({ [`v0/consortium/${tokenId}/members`]: infoObjs })
+  }
+
+  async function setup (): Promise<void> {
+    accountAlice = await alice.generateAddress()
+    accountBob = await bob.generateAddress()
+
+    await alice.token.create({
+      symbol: symbolBTC,
+      name: symbolBTC,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: accountAlice
+    })
+    await alice.generate(1)
+
+    await alice.token.create({
+      symbol: symbolETH,
+      name: symbolETH,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: accountAlice
+    })
+    await alice.generate(1)
+
+    idBTC = await alice.token.getTokenId(symbolBTC)
+    idETH = await alice.token.getTokenId(symbolETH)
+
+    await setGovAttr({
+      'v0/params/feature/consortium': 'true',
+      [`v0/consortium/${idBTC}/mint_limit`]: '20',
+      [`v0/consortium/${idBTC}/mint_limit_daily`]: '10',
+      [`v0/consortium/${idETH}/mint_limit`]: '40',
+      [`v0/consortium/${idETH}/mint_limit_daily`]: '20'
+    })
+
+    await setMemberInfo(idBTC, [{
+      id: '01',
+      name: 'alice',
+      ownerAddress: accountAlice,
+      backingId: 'abc',
+      mintLimitDaily: '10',
+      mintLimit: '20'
+    }, {
+      id: '02',
+      name: 'bob',
+      ownerAddress: accountBob,
+      backingId: 'def,hij',
+      mintLimitDaily: '10',
+      mintLimit: '20'
+    }])
+
+    await setMemberInfo(idETH, [{
+      id: '01',
+      name: 'alice',
+      ownerAddress: accountAlice,
+      backingId: '',
+      mintLimitDaily: '20.00000000',
+      mintLimit: '40.00000000'
+    }, {
+      id: '02',
+      name: 'bob',
```

Comment:
I wonder, any reason we return the name?

Reply:
Yeah, we're showing member name in the FE.